<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddCoursewareWindow</class>
 <widget class="C8PopupWindow" name="AddCoursewareWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>344</width>
    <height>132</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AddCoursewareWindow</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="bottomMargin">
      <number>10</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_localFile">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_skyFile">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_whiteBoard">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>C8PopupWindow</class>
   <extends>QWidget</extends>
   <header>c8popupwindow.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_skyFile</sender>
   <signal>clicked()</signal>
   <receiver>AddCoursewareWindow</receiver>
   <slot>addSkyFileBtnClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>173</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>26</x>
     <y>66</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_localFile</sender>
   <signal>clicked()</signal>
   <receiver>AddCoursewareWindow</receiver>
   <slot>addLoacalFileBtnClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>87</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>64</x>
     <y>62</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_whiteBoard</sender>
   <signal>clicked()</signal>
   <receiver>AddCoursewareWindow</receiver>
   <slot>addWhiteBoardBtnClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>260</x>
     <y>64</y>
    </hint>
    <hint type="destinationlabel">
     <x>103</x>
     <y>54</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addSkyFileBtnClicked()</slot>
  <slot>addLoacalFileBtnClicked()</slot>
  <slot>addWhiteBoardBtnClicked()</slot>
 </slots>
</ui>
